From: Matthias Klose <doko@debian.org>
Date: Wed, 19 Apr 2017 12:25:35 -0700
Subject: locale-module

# DP:   * Lib/locale.py:
# DP:     - Don't map 'utf8', 'utf-8' to 'utf', which is not a known encoding
# DP:       for glibc.
---
 Lib/locale.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Lib/locale.py b/Lib/locale.py
index e85f34e..50864ec 100644
--- a/Lib/locale.py
+++ b/Lib/locale.py
@@ -1385,8 +1385,8 @@ locale_alias = {
     'ug_cn':                                'ug_CN.UTF-8',
     'uk':                                   'uk_UA.KOI8-U',
     'uk_ua':                                'uk_UA.KOI8-U',
-    'univ':                                 'en_US.utf',
-    'universal':                            'en_US.utf',
+    'univ':                                 'en_US.UTF-8',
+    'universal':                            'en_US.UTF-8',
     'universal.utf8@ucs4':                  'en_US.UTF-8',
     'unm_us':                               'unm_US.UTF-8',
     'ur':                                   'ur_PK.CP1256',
